maps-app/build.gradle

```diff
@@ -28,15 +28,16 @@ android {
 
 dependencies {
     // android support libraries
-    compile 'com.android.support:support-v4:25.0.1'
-    compile 'com.android.support:appcompat-v7:25.0.1'
-    compile 'com.android.support:cardview-v7:25.0.1'
-    compile 'com.android.support:design:25.0.1'
+    implementation 'com.android.support:support-v4:27.0.2'
+    implementation 'com.android.support:appcompat-v7:27.0.2'
+    implementation 'com.android.support:cardview-v7:27.0.2'
+    implementation 'com.android.support:design:27.0.2'
 
     // ArcGIS Android API
-    compile 'com.esri.arcgisruntime:arcgis-android:100.1.0'
+    implementation 'com.esri.arcgisruntime:arcgis-android:100.2.0'
     // square libs
-    compile 'com.jakewharton:butterknife:6.1.0'
+    implementation 'com.jakewharton:butterknife:8.8.1'
+    annotationProcessor 'com.jakewharton:butterknife-compiler:8.8.1'
 
     // Android Testing Support Library's runner and rules
     androidTestCompile ("com.android.support.test:runner:$rootProject.ext.runnerVersion"){
```

Comment:
I think all the androidTestCompiles should be replaced with androidTestImplementation